app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  # logs in the user
+  def log_in(user)
+    session[:user_id] = user.id
```

Comment:
Don't use `session`

- Generate and Update the user `remember_token` here
- And then call you `remember(user)` method